### PR TITLE
Fix of non thread safe map reqHeaders

### PR DESCRIPTION
**Pull request by vbnhfylbh**
SOAP_ACTION is shared between different requests despite thread.local.request.context=true. Reproduced under load of 100 transactions per second.

### CODE REVIEW

File: rt/bindings/soap/src/main/java/org/apache/cxf/binding/soap/interceptor/SoapPreProtocolOutInterceptor.java [hunk 1]
```diff
@@ -19,24 +19,16 @@
...
-import java.util.List;
-import java.util.Map;
-import java.util.TreeMap;
-
-import org.apache.cxf.binding.soap.Soap11;
-import org.apache.cxf.binding.soap.Soap12;
-import org.apache.cxf.binding.soap.SoapBindingConstants;
-import org.apache.cxf.binding.soap.SoapMessage;
-import org.apache.cxf.binding.soap.SoapVersion;
+import org.apache.cxf.binding.soap.*;
```
Comment (deki) [inline rt/bindings/soap/src/main/java/org/apache/cxf/binding/soap/interceptor/SoapPreProtocolOutInterceptor.java]: please avoid star imports

File: rt/bindings/soap/src/main/java/org/apache/cxf/binding/soap/interceptor/SoapPreProtocolOutInterceptor.java [hunk 2]
```diff
@@ -19,24 +19,16 @@
...
 import org.apache.cxf.binding.soap.model.SoapOperationInfo;
 import org.apache.cxf.helpers.CastUtils;
 import org.apache.cxf.interceptor.Fault;
 import org.apache.cxf.message.Message;
 import org.apache.cxf.phase.Phase;
 import org.apache.cxf.service.model.BindingOperationInfo;
 
+import java.util.*;
```
Comment (deki) [inline rt/bindings/soap/src/main/java/org/apache/cxf/binding/soap/interceptor/SoapPreProtocolOutInterceptor.java]: please avoid star imports

**Comment by vbnhfylbh** [comment]
Thanks for your feedback. Created https://github.com/apache/cxf/pull/306

**Comment by vbnhfylbh** [comment]
@deki, thanks for your feedback. Please review

**Comment by deki** [comment]
Looks like you need to rebase.

**Pull request closed without merging**